web/src/routes/backOffice/employees/components/EmployeeForm.svelte

```diff
@@ -26,10 +30,37 @@
 	 */
 	export let title: string;
 
+	/**
+	 * Set form as a create form, to create a new employee.\
+	 * @default false
+	 */
+	export let createForm: boolean = false;
+
 	/**
 	 * Callback fired when save action is triggered.
 	 */
-	export let onSave: (
+	export let onSave: onSaveType | onSaveCreateType;
+
+	/**
+	 * Type of callback to create new employee.
+	 */
+	type onSaveCreateType = (
```

Comment:
```suggestion
	type onSaveCreateFn = (
```
Avoid using "type" for naming types